client/web/src/site-admin/AccessRequestsPage/index.tsx

```diff
@@ -225,21 +232,48 @@ export const AccessRequestsPage: React.FunctionComponent = () => {
                         resetPasswordURL={lastApprovedUser.resetPasswordURL}
                     />
                 )}
-                <div className="d-flex justify-content-end">
-                    <PageSwitcher
-                        totalCount={connection?.totalCount ?? null}
-                        totalLabel={connection?.totalCount === 1 ? 'account request' : 'account requests'}
-                        {...paginationArgs}
-                    />
+                <div className="d-flex align-items-start justify-content-between">
+                    <AccessRequestStatusPicker status={status as AccessRequestStatus} onChange={setStatus} />
+                    <div className="d-flex justify-content-end mt-4">
+                        <PageSwitcher
+                            totalCount={connection?.totalCount ?? null}
+                            totalLabel={connection?.totalCount === 1 ? 'account request' : 'account requests'}
```

Comment:
[`pluralize`](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/common/src/util/strings.ts?L24-26) can be used here
```suggestion
                            totalLabel={pluralize('account request', connection?.totalCount || 0)}
```